UI.py

```diff
@@ -211,7 +211,63 @@ def CreateIsMemberWorkingOptions():
     ]
 
     return member_options
+
+def CreateGetEndOfMonthCalculationsOptions():
+    end_of_month_calculations_options = [
+        create_option(
+            name = "month",
+            description = "number of the desired month (optional) leave empty for current month",
+            option_type = SlashCommandOptionType.INTEGER,
+            required = False
+        ),
+        create_option(
+            name = "year",
+            description = "number of the desired year (optional) leave empty for current year",
+            option_type = SlashCommandOptionType.INTEGER,
+            required = False
+        )
+    ]
+
+    return end_of_month_calculations_options
     
+def CreateGetEndOfMonthCalculationsEmbed(month = None, year = None):
+    month = month or datetime.datetime.now().month
+    year = year or datetime.datetime.now().year
+    embed = discord.Embed(
+        title = f'End of Month Calculations',
+        description = f'{month}/{year} Calculations:',
+        colour = 0x4682B4
+    )
+    embed.set_thumbnail(url = os.getenv("Salary_Image"))
+    embed.add_field(name = '\u200B', value = '\u200B', inline = False)
+
+    for member in member_db.GetMembers():
+        member_data = FormatGetEndOfMonthCalculationsEmbed(member, month, year)
+        if member_data == "":
+            continue
+        member_name = member_db.GetMemberByID(member["id"])["name"]
+        embed.add_field(name = f'**{member_name.upper()}**', value = member_data, inline = False)
+        embed.add_field(name = '\u200B', value = '\u200B', inline = False)
```

Comment:
should take in a list of members

Reply:
i dont understand what is the difference, you trying to achieve but passing list rather than calling `member_db.GetMembers()`

Reply:
Done